code/modules/cargo/expressconsole.dm

```diff
@@ -86,11 +116,28 @@
 		packin_up()
 		stack_trace("You didn't give the cargo tech good advice, and he ripped the manifest. As a result, there was no pack data for [src]")
 	data["supplies"] = meme_pack_data
-
+	if (cooldown > 0)
+		cooldown--
 	return data
 
 /obj/machinery/computer/cargo/express/ui_act(action, params, datum/tgui/ui)
 	switch(action)

```

Comment:
does this fail is the mob is dead? don't see any canUseTopic checks anywhere

Reply:
@ShizCalev yes it does, because its a subtype of console. (just tested it)

Reply:
tgui will handle it with the default interaction type iirc.